packages/eslint-config/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     'plugin:prettier/recommended',
     'plugin:react-hooks/recommended',
   ],
-  plugins: ['prettier', 'tailwindcss', 'react-hooks'],
+  plugins: ['prettier', 'tailwindcss', 'react-hooks', 'unused-imports'],
```

Comment:
Added this so I can autofix imports after refactor, we have it in API as well.

Reply:
I both love and hate that... so many times I've temporarily comment out some code, then my editor autofixed lint issues and removed whatever the commented out code had imported :triggered:

Reply:
I can see that, but most of the time I think it speeds up. I am open to removing it as I literally needed it to fix stuff in this PR 😄

Reply:
I'm fine with keeping it, it's just _sometimes_ a bit annoying to have to re-import things 😅 Not that it's super difficult.
And often it'a nice to have, as you remove code, but forgot to remove the import. We just need an AI version of the plugin, so it knows when to remove it and not 😂